src/backend/magnify/settings.py

```diff
@@ -200,6 +200,9 @@ class Base(Configuration):
         "jitsi_app_id": values.Value(
             "", environ_name="JWT_JITSI_APP_ID", environ_prefix=None
         ),
+        "jitsi_xmpp_domain": values.Value(
+            "", environ_name="JWT_JITSI_XMPP_DOMAIN", environ_prefix=None
```

Comment:
I would make it required unless it can work with this setting set to an empty string.